### PR TITLE
Issue 31: Rerun PA as a Proxy Analysis with 2018 Data

### PR DESCRIPTION
**Pull request by mikemahoney218**
This commit contains code and analyses for Issue #31: Rerun PA as a Proxy Analysis with 2018 Data. Using this new dataset, I found that PA is an _ok_ fit for MA crime rates (at least according to this 2018 arrest rate data); however, other states (including NJ, KY) are closer fits. More discussion on that issue page.

### CODE REVIEW

**Comment by mikemahoney218** [comment]
sorry about the hella commit history on this branch; this feels like a good time to use that "squash & merge" button :sweat_smile:

**Comment by mikemahoney218** [comment]
also -- and this might be a question for @dawngraham -- I'm not sure if I should save files of the datasets I use for this actual analysis; I never saved them as objects since there were only about 3 steps between the cleaned data and the input I needed

File: analyses/determine_closest_state_2018_fbi/repeat_past_analysis.Rmd [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+output: pdf_document
+---
+
+So first things first, let's load our libraries and data:
```
Comment (mikemahoney218) [inline analyses/determine_closest_state_2018_fbi/repeat_past_analysis.Rmd:6]: add documentation header (yes I know this is my own commit, I just broke my own rules)

File: data/cleaned/fbi_aggregated_2018/fbi_aggregated_2018.Rmd [hunk 2]
```diff
@@ -0,0 +1,36 @@
+This script cleans the FBI 2018 crimes dataset, fixing column names and blank cells.
```
Comment (mikemahoney218) [inline data/cleaned/fbi_aggregated_2018/fbi_aggregated_2018.Rmd]: specify inputs and outputs

**Comment by mikemahoney218** [comment]
I have to add a readme in my project folder and document both new scripts a bit before I can really merge this -- you'd figure I'd be more cognizant of that, given I wrote the dang rule. Should fix by Tuesday

**Comment by dawngraham** [comment]
@mikemahoney218 I probably did not review this as closely as I could/should have before merging. I did catch a broken link and what look to be unnecessary files - will create new issues.

**Pull request merged**